app/components/preNoted/suddenAlert.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from "react";
+import Popper, { PopperProps } from "@material-ui/core/Popper";
+import { withStyles } from "../../helpers/withStylesHelper";
+import { useIntervalProgress } from "../../hooks/useIntervalProgressHook";
+const styles = require("./suddenAlert.scss");
+
+interface SuddenAlertProps extends PopperProps {}
+
+const SuddenAlert: React.FC<SuddenAlertProps> = props => {
+  const [renderTime, setRenderTime] = React.useState(0);
+  const [shouldShowSuddenAlert, setShouldShowSuddenAlert] = React.useState(false);
+
+  useIntervalProgress(() => {
```

Comment:
you don't need to use `setInterval` for this.
why don't you just use `setTimeout` instead?

Reply:
Thanks for a good suggestion.